dashboard/src/App.vue

```diff
@@ -89,6 +85,7 @@ body {
 
 main {
 	flex: 1;
+	background-color: rgba($color: #fff, $alpha: 0.1);
```

Comment:
I believe we could import and use variables from the theme here, similar to how its done here https://github.com/wolfiabot/Wolfia/blob/master/dashboard/src/components/Togglz.vue#L42 and here https://github.com/wolfiabot/Wolfia/blob/master/dashboard/src/components/Togglz.vue#L49

Reply:
Fair point.